roles/ceph-mon/tasks/deploy_monitors.yml

```diff
@@ -63,7 +63,7 @@
 - name: ceph monitor mkfs with keyring (before infernalis release)
   command: ceph-mon --mkfs -i {{ monitor_name }} --fsid {{ fsid }} --keyring /var/lib/ceph/tmp/keyring.mon.{{ monitor_name }}
```

Comment:
we don't use it in ceph-docker AFAIR it is not needed but I'm currently validating this :)